script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -213,6 +214,8 @@ I can't stand the thought of him 'existing' for even a second longer.
 [sha:8620bac62396edeea2cf1a5642460e64e09ab2b4]{
 -- Page 416, Offset 32857.
 -- まさか、その体で私とやりあおうというハラでもあるまい。
+// group pr check:retl
+// suggestions: There's nothing that makes me less excited than pointless struggles. or There's nothing more ugly than a pointless struggle.
```

Comment:
I like There's nothing more ugly than a pointless struggle.

Reply:
I would substitute ugly with appalling, just for added impact.

Reply:
Am I missing something? This seems like a fairly liberal change

Reply:
this was a Petrikow suggestion, I don't really remember the specifics anymore.

Reply:
Oh, probably because this edit was applied to the wrong line. It's for the next line at offset 32858

Reply:
Ok that makes a lot more sense